assets/dev/js/editor/document/globals/commands/disable.js

```diff
@@ -0,0 +1,21 @@
+import DisableEnable from './base/disable-enable';
+
+export class Disable extends DisableEnable {
+	apply( args ) {
+		const { settings, containers = [ args.container ] } = args;
+
+		containers.forEach( ( container ) => {
+			container = container.lookup();
+
+			Object.keys( settings ).forEach( ( setting ) => {
+				container.globals.unset( setting );
+			} );
+
+			container.settings.set( '__globals__', container.globals.toJSON() );
+
+			container.render();
```

Comment:
Why do we need a full render for the container when the global being disabled?

Reply:
It's temporary, until it will work with CSS varibales.